src/librustc_mir/build/expr/into.rs

```diff
@@ -234,8 +252,8 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
                         is_block_tail: None,
                     });
                     let ptr_temp = Place::from(ptr_temp);
-                    let block = unpack!(this.into(&ptr_temp, block, ptr));
-                    this.into(&ptr_temp.deref(), block, val)
+                    let block = unpack!(this.into(&ptr_temp, None, block, ptr));
```

Comment:
These "no drop scopes" aren't commented, though they may be obvious enough due to raw pointers not needing drop